### PR TITLE
[IOPID-2011] integrate setting pictogram

### PR DESCRIPTION
**Pull request by Ladirico**
## Short description
This PR introduces the `PictogramSettings` and `PictogramBleedSettings` pictograms.

Screenshot 2024-07-10 alle 12 45 01

## How to test
Within the example app, navigate to the **Pictograms** section and check that the new pictograms are correctly rendered

### CODE REVIEW

**Review by dmnplb — CHANGES_REQUESTED** [review]
The pictogram you need is part of the `Bleed` group. In general when we have a new pictogram, there are two versions:
- Standard (the one you can see in the bigger blocks in the **Pictograms** screen)
- Bleed (the one you can see in the smaller blocks below, in the same screen)

If we follow this file organization, the exported pictogram should be named `PictogramBleedSettings`. And the standard should have the filename `PictogramSettings`.

Everything is already explained [in this document](https://github.com/pagopa/io-app-design-system/blob/main/src/components/pictograms/README.md#add-a-new-pictogram).

**tldr:** just export from Figma `PictogramSettings.svg` and `PictogramBleedSettings.svg`, place them in the `svg/originals` folder and run the command `yarn generate:pictograms`. Remember to add the keys to the `Pictograms` object and relative `Bleed` subobject.

**Comment by Ladirico** [comment]
> The pictogram you need is part of the `Bleed` group. In general when we have a new pictogram, there are two versions:
> 
> * Standard (the one you can see in the bigger blocks in the **Pictograms** screen)
> * Bleed (the one you can see in the smaller blocks below, in the same screen)
> 
> If we follow this file organization, the exported pictogram should be named `PictogramBleedSettings`. And the standard should have the filename `PictogramSettings`.
> 
> Everything is already explained [in this document](https://github.com/pagopa/io-app-design-system/blob/main/src/components/pictograms/README.md#add-a-new-pictogram).
> 
> **tldr:** just export from Figma `PictogramSettings.svg` and `PictogramBleedSettings.svg`, place them in the `svg/originals` folder and run the command `yarn generate:pictograms`. Remember to add the keys to the `Pictograms` object and relative `Bleed` subobject.

Thanks for the correction. I have made a new push with the corrections! @dmnplb

**Review by dmnplb — APPROVED** [review]
